profiles/base.yaml

```diff
@@ -608,3 +608,8 @@ discard_scrolls:
 - Seal Cambrinth
 - Strange Arrow
 
+theurgy_whitelist:
+theurgy_blacklist:
+- embarass_myself
+theurgy_exp_threshold: 0
+theurgy_scholarship_threshold: 34
```

Comment:
Should this be removed?

Reply:
Yes, sorry -- forgot to include base.yaml in this update.